src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/custom-anomalies.mdx

```diff
@@ -0,0 +1,104 @@
+
+import alertsAnomaliesSetUpperandLowerRanges from 'images/alerts_screenshot-full_anomalies-set-upper-and-lower-ranges.png'
+
+import alertsFacetedAnomaliesTwo from 'images/alerts_screenshot-full_faceted-anomalies-two.png'
+
+import alertsFacetedAnomalies from 'images/alerts_screenshot-full_faceted-anomalies.png'
+
+import alertsConfigureYourAnomalyThresholds from 'images/alerts_screenshot-full_configure-your-anomaly-thresholds.png'
+
+import iconAlertCritical from 'images/icon-alert-critical.png'
+
+import iconAlertWarning from 'images/icon-alert-warning.png'
+
+Custom anomalies allow your team the most flexibility when detecting unusual behavior in your system. Not only are they flexible and dynamic, custom anomalies provide your team with the capability to alert on any NQRL condition and to adjust and optimize your thresholds. Custom anomalies are built using the same advanced tuning settings as static alerting so you can ensure your team sees only incidents that are important to you. 
+
+To create a custom anomaly, first you must create a condition. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Alerts & AI > Policies > (create or select policy) > Create alert condition**: You can select APM, Browser, or NRQL condition. 
+
+Then choose anomaly as your alert condition threshold type.
+
+This threshold will predict the future behavior for the value of your data. . This prediction appears as a dotted black line on the preview chart in the anomaly condition UI. 
+
+<img
+  title="How custom anomalies work"
+  alt="A screenshot demonstrating active custom anomalies"
+  src={alertsHowCustomAnomaliesWork}
+/>
+
+### Configure anomaly thresholds
+
+After you select 'anomaly' for threshold type, you'll need to set your threshold. This will set the criteria that you want to be notified on. 
+
+To do this you'll need to:
+* Set whether you want the condition to check for upper violations, lower violations, or both.
+* Set the behavior that will trigger an incident (for example: "deviating for more than five minutes") 
+* Adjust how sensitive the condition is to fluctuations in the data source by adjusting the sliding bar to more violations or fewer violations. This bar calculates how many standard deviations away from the prediction  you want to be notified.
+
+<img
+  title="Custom anomaly configuration"
+  alt="A screenshot demonstrating how to configure custom anomalies"
+  src={alertsConfigureYourAnomalyThresholds}
+/>
+
+When your data escapes the predicted "normal" behavior and meets the criteria you've chosen, you'll receive a notification.
+
+Here are some tips for setting anomaly thresholds:
+
+* Set the [anomaly direction](#upper-lower) to monitor violations that happen either above or below the anomaly.
+* Set the preview chart to either [2 days or 7 days](#days) of displayed data. (Not applicable for [NRQL alert conditions](/docs/alerts/new-relic-alerts/defining-conditions/create-alert-conditions-nrql-queries).)
```

Comment:
I think you can just tweak the sentence and eliminate the parentheses.

Reply:
done